presentation/src/main/java/com/example/healthysalad/TitleFragment.kt

```diff
@@ -14,12 +14,28 @@ class TitleFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         var v : View = inflater.inflate(R.layout.fragment_title,container, false)
+
+        val createSalad = SelectIngredientFragment.newInstance("base")
```

Comment:
Please name the variable understandable.
It is a fragment, so at least you should name it as CreateSaladFragment. 
I think the best name is a selectIngredientFragment, but why you choose this name?

Reply:
because it's the start of the create salad flow, I've changed the name to you suggested one. it makes more sense